server/src/test/resources/sql/materialized_view.iq

```diff
@@ -39,12 +39,12 @@ select * from dept where deptno > 10;
 
 # Check contents
 select * from v;
-+--------+-------------+
-| DEPTNO | NAME        |
-+--------+-------------+
-|     20 | Marketing   |
-|     30 | Engineering |
```

Comment:
The changes here are a bit strange. Maybe we can change the table definition statement.
```
# Create a source table
create table dept (deptno int not null, name varchar(10));
(0 rows modified)

!update

insert into dept
values (10, 'Sales'), (20, 'Marketing'), (30, 'Engineering');
(3 rows modified)
```
According to the output content, was cropping done during output or input？

Reply:
I actually don't know how insert is implemented in Calcite.
But no matter what, the result is now correct, and it wasn't before. 
So I think this is actually a good test.

Reply:
Ok. I get it.